pom.xml

```diff
@@ -441,10 +441,6 @@
             <goals>
               <goal>check</goal>
             </goals>
-            <configuration>
-              <failOnViolation>true</failOnViolation>
```

Comment:
Do we need to delete this line? line 428 is like <failOnViolation>false</failOnViolation>.
How about we just delete <excludes>com/linkedin/tony/rpc/proto/*,com/linkedin/tony/events/*</excludes>